components/brave_wallet/browser/BUILD.gn

```diff
@@ -277,3 +289,10 @@ source_set("transaction") {
     "//brave/components/json/rs:cxx",
   ]
 }
+
+config("sardine_config") {
+  defines = [
+    "SARDINE_CLIENT_ID=\"$sardine_client_id\"",
+    "SARDINE_CLIENT_SECRET=\"$sardine_client_secret\"",
```

Comment:
@Nick-Love21 please convert this into buildflags. See for example https://github.com/brave/brave-core/pull/12928